docusaurus.config.js

```diff
@@ -136,25 +239,67 @@ const config = {
         },
         items: [
           {
-            href: 'https://oasisprotocol.org/',
-            label: 'Home',
+            label: 'General',
+            type: 'doc',
+            position: 'left',
+            docId: 'README',
+          },
+          {
+            label: 'Node Operators',
+            to: '/operators/',
+            activeBaseRegex: '/operators/',
             position: 'left',
           },
           {
-            href: 'https://discord.gg/RwNTK8t',
-            label: 'Support',
+            label: 'Developers', // dApps, ParaTimes
+            to: '/developers/',
+            activeBaseRegex: '/developers/',
             position: 'left',
           },
+          {
+            label: 'Core', // oasis-core, ADRs
+            to: '/core/',
+            position: 'left',
+            activeBaseRegex: `/core/`,
+          },
           {
             href: 'https://github.com/oasisprotocol',
             position: 'right',
             className: 'header-github-link',
+            title: 'GitHub repository',
             'aria-label': 'GitHub repository',
           },
+          {
+            href: 'https://oasisprotocol.org/',
+            position: 'right',
+            className: 'header-www-link',
+            title: 'Oasis Protocol Foundation website',
+            'aria-label': 'Oasis Protocol Foundation website',
+          },
         ],
       },
       footer: {
         style: 'dark',
+        links: [
+          {
+            title: 'Support',
+            items: [
+              {
+                label: 'Discord',
+                href: 'https://discord.gg/RwNTK8t',
```

Comment:
This is the old broken link

Reply:
After all discord invites are pJdWeVtmHT we can validate if invite has expired by e.g.

- adding to package.json
```json
  "blc": "linkinator http://localhost:3000/general https://discord.com/api/v9/invites/pJdWeVtmHT",
```

- or adding to markdown somewhere
```md
[](https://discord.com/api/v9/invites/pJdWeVtmHT)
```

Reply:
Thanks, fixed.